### ISSUE TITLE
eli5 determining outbound ips

### ISSUE THREAD

**Issue by Datise** [segmentio/stack#143]
I'd like to start by saying thanks for this work and making it open source. It's awesome. Also, sorry if this isn't the right place for a question like this.

background:  
I'm trying to scaffold an infrastructure with multiple services inside one of the stacks' ecs clusters (two external facing, one internal only). All of the services are being tucked behind/registered to a load balancer. All the load balancers are given their own subdomain in route 53. 

current understanding:
My understanding from the readme (and this line "The VPC itself will contain a single network gateway to route traffic in and out of the different subnets.") is that each ecs cluster will have its own nat gateway with private and public CIDR, and that everything will be routed through a single internet gateway. 

question: 
Given my undertanding:
1. does an ecs cluster have a single elastic (static) IP address provided by the internet gateway, that I should be able to give to an external org to be whitelisted.
  - where should I be able to find that on an amazon dashboard? (I've looked at the ec2 elastic ip and network gateway pages but I'm finding it difficult to know for sure) 

2. If its not a single one, is there an outbound range I can depend on?

**Comment by raam86** [comment]
@Datise You mentioned you put all instances behind an ELB. That elb should have a static address that you should be able to distribute
